k8s_test_harness/util/docker_util.py

```diff
@@ -1,11 +1,12 @@
 #
-# Copyright 2024 Canonical, Ltd.
+# Copyright 2025 Canonical, Ltd.
 # See LICENSE file for licensing details
 #
 
 import json
 import logging
 import os
+import shlex
 import stat
 import subprocess
 from typing import List
```

Comment:
For improved typing below

```suggestion
from typing import List, Optional, Union
```